.vscode/settings.json

```diff
@@ -8,5 +8,6 @@
     },
     "[html]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
-    }
+    },
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
should we add `"typescript.enablePromptUseWorkspaceTsdk": true` to prompt folks? otherwise lgtm

Reply:
That's a good idea! Didn't know about that option. Going to add it :)

Reply:
Added!